cmd/csi_driver/main.go

```diff
@@ -19,15 +19,28 @@ package main
 import (
 	"context"
 	"flag"
+	"fmt"
+	"log"
+	"net"
+	"os"
+	"os/exec"
+	"strconv"
+	"strings"
 
 	"github.com/GoogleCloudPlatform/lustre-csi-driver/pkg/cloud_provider/lustre"
 	"github.com/GoogleCloudPlatform/lustre-csi-driver/pkg/cloud_provider/metadata"
 	driver "github.com/GoogleCloudPlatform/lustre-csi-driver/pkg/csi_driver"
 	"github.com/GoogleCloudPlatform/lustre-csi-driver/pkg/metrics"
+	"github.com/vishvananda/netlink"
```

Comment:
I think we should move these multi-NIC–specific functions into a separate package under `/pkg`, outside of `main.go`.